### PR TITLE
added outline secondary button to demo page

### PR DESCRIPTION
**Pull request by julieduong22**

### CODE REVIEW

**Review by cwolf10 — COMMENTED** [review]
Can we add a demo page for this variant to show that it's an option?

**Comment by julieduong22** [comment]
How should the disable outline secondary button look like? Should it look like the outline button or just like it looks right now?

File: src/stylesheets/elements/_buttons.scss [hunk 1]
```diff
@@ -80,6 +80,37 @@
   background-color: transparent;
   box-shadow: none;
 
+  // Add color variants here
+  &.usa-button--secondary {
+  	@include u-border("secondary");
+    background-color: white;
```
Comment (davereed) [inline src/stylesheets/elements/_buttons.scss:86]: The mockup shows semi-bold text in the "ink" color, so we need to add:
@include u-text('semi-bold', 'ink');

File: src/stylesheets/elements/_buttons.scss [hunk 2]
```diff
@@ -80,6 +80,37 @@
...
+
+    &:visited {
+      color: black;
+      box-shadow: none;
+    }
+
+    &.usa-button--hover,
+    &:hover {
+      background-color: color("secondary-lighter");
```
Comment (davereed) [inline src/stylesheets/elements/_buttons.scss:95]: Just FYI, this doesn't match the mockup. But the mockup uses a non-theme (invalid) color, so we need to make Shannon/Designers aware of this issue that we had to substitute it.

**Review by davereed — CHANGES_REQUESTED** [review]
Add the u-text change and this should be good.

File: src/stylesheets/elements/_buttons.scss (hunk 2 above)
Comment (julieduong22) [reply src/stylesheets/elements/_buttons.scss:95]: I see. I'll tell Shannon tomorrow.

File: src/stylesheets/elements/_buttons.scss [hunk 3]
```diff
@@ -80,6 +80,38 @@
   background-color: transparent;
   box-shadow: none;
 
+  // Add color variants here
+  &.usa-button--secondary {
+  	@include u-border("secondary");
```
Comment (cwolf10) [inline src/stylesheets/elements/_buttons.scss:85]: Can you fix the indentation here?

File: src/stylesheets/elements/_buttons.scss (hunk 2 above)
Comment (cwolf10) [reply src/stylesheets/elements/_buttons.scss:96]: I haven't seen that mockup, but are we sure we want to color the background on the non-inverse outline button? The primary outline just uses transparent as its background. Inverse uses a colored background. Should this class really be an inverse variant?

**Review by davereed — APPROVED** [review]

**Pull request merged**